jtelegrambotapi-core/src/main/java/com/jtelegram/api/message/entity/MessageEntity.java

```diff
@@ -14,4 +22,20 @@
     public void updateContent(String text) {
         this.content = text.substring(offset, offset + length);
     }
+
+    static class DefaultMessageEntity extends MessageEntity {}
+    public static class Deserializer implements JsonDeserializer<MessageEntity> {
+
+        @Override
+        public MessageEntity deserialize(JsonElement json, Type typeOfT, JsonDeserializationContext context) throws JsonParseException {
+            JsonObject object = json.getAsJsonObject();
+            MessageEntityType type = context.deserialize(object.get("type"), MessageEntityType.class);
+            if (type == null) {
+                throw new UnsupportedOperationException("This version of jTelegramBotAPI does not support '" + object.get("type").getAsString() + "' message entities.");
```

Comment:
Is throwing an exception in a deserializer the best idea here? It essentially breaks the program and doesn't allow functionality of the API if a developer is using an outdated version. Possibly returning null with a log/warning would be best in such an event.

Reply:
The other option is to have an `MessageEntityType#UNSUPPORTED` and use that.

Reply:
Either one works, just so long that there is a message in console indicating why the behavior of the bot would be abnormal.